Doc/c-api/hash.rst

```diff
@@ -41,6 +45,28 @@ See also the :c:member:`PyTypeObject.tp_hash` member.
    .. versionadded:: 3.4
 
 
+Functions
+^^^^^^^^^
+
+.. c:function:: int Py_HashDouble(double value, Py_hash_t *result)
+
+   Hash a C double number.
+
+   * Set *\*result* to the hash and return ``1`` if *value* is finite or is
+     infinity.
+   * Set *\*result* to :data:`sys.hash_info.nan <sys.hash_info>` (``0``) and
```

Comment:
Is it necessary/desirable to document that `sys.has_info.nan` is 0?  ``_PyHASH_NAN`` is defined with a leading underscore after all.

Reply:
I rephrased the doc to no longer mention NaN directly.